src/components/organisms/MediaBox.jsx

```diff
@@ -48,6 +60,31 @@ export const MediaBox = ({ media, isPreviewMobile }) => {
     Player = DeezerPlayer
   }
 
+  let PlayerWrapper =
+    (!!media?.fullscreen && Fullscreen) ||
+    (Player === ReactPlayer && AutoPlayerFit) ||
+    (Player !== ReactPlayer &&
+      media?.deezer?.format === 'SQUARE' &&
+      !isPreviewMobile &&
+      AutoPlayerFit) ||
+    Wrapper
+
+  // return (
```

Comment:
Please remove commented code or leave a clear message in the code explaining the reason the code is commented.